.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and test
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        rust: [stable, beta, nightly]
```

Comment:
Leaving `stable` and `nightly` will be sufficient for us, `beta` is too bleeding-edge :D

Reply:
If I understand [this](https://rust-lang.github.io/rustup/concepts/channels.html
) correctly nightly is the bleeding-edge, isn't it?

Reply:
Oh, that's right, I am sorry.

Anyway, let's keep `stable` and `nightly`, `beta` is barely used in CI :>